docs/cli/commands.md

```diff
@@ -0,0 +1,561 @@
+declarations for the package, and also requires `yarn tsc` to have been run
+first.
+
+```text
+Usage: backstage-cli build [options]
+
+Options:
+  --outputs &lt;formats&gt;  List of formats to output [types,cjs,esm]
+  -h, --help           display help for command
+```
+
+## lint
+
+Scope: `any`
+
+Lint a package. In addition to the default `eslint` behavior, this command will
+include TypeScript files, treat warnings as errors, and default to linting the
+entire directory of no specific files are listed.
+
+```text
+Usage: backstage-cli lint [options]
+
+Options:
+  --format &lt;format&gt;  Lint report output format (default: "eslint-formatter-friendly")
+  --fix              Attempt to automatically fix violations
+  -h, --help         display help for command
+```
+
+## test
+
+Scope: `any`
+
+Run tests, forwarding all unknown options to Jest, and defaulting to watch mode.
+When executing the tests, `process.env.NODE_ENV` will be set to `"test"`.
+
+This command uses a default Jest configuration that is included in the CLI,
+which is set up with similar goals for speed, scale, and working within a
+monorepo. The configuration sets the `src` as the root directory, enforces the
+`.test.` infix for tests, and uses `src/setupTests.ts` as the test setup
+location.
+
+If needed, the configuration can be extended using a `"jest"` field in
+`package.json`, both within the target package and the monorepo root, with
+configuration in the target package taking precedence. Refer to the
+[Jest configuration](https://jestjs.io/docs/en/configuration) for a full list of
+configuration options.
+
+In addition to the Jest configuration there's an optional `transformModules`
+option, which is an array of module names to include in transformations.
+Normally modules inside `node_modules` are not transformed, but there are cases
+were published packages are not transpiled far enough to be usable by Jest, in
+which case you need to enable transform of them.
+
+Another way to override the Jest configuration is to place a `jest.config.js` or
+`jest.config.ts` file in the package root. As opposed to the `package.json` way
+of overriding config, this completely removes the base config, and so you need
+to set it up from scratch.
+
+```text
+Usage: backstage-cli test [options]
+
+Options:
+  --backstage-cli-help    display help for command
+```
+
+## config:print
+
+Scope: `root`
+
+Print the static configuration, defaulting to reading `app-config.yaml` in the
+repo root, using schema collected from all local packages in the repo.
+
+For example, to validate that a given configuration value is visible in the
+frontend when building the `my-app` package, you can use the following:
+
+```bash
+yarn backstage-cli config:print --frontend --package my-app
+```
+
+```text
+Usage: backstage-cli config:print [options]
+
+Options:
+  --package &lt;name&gt;   Only load config schema that applies to the given package
+  --frontend         Print only the frontend configuration
```

Comment:
maybe link to the config section explaining visibility options

Reply:
can't :<